javascript/app.js

```diff
@@ -0,0 +1,254 @@
+
+// What are the implications of the ability of a function to return a value?
+// return being able to return a value implies one will simply be able to invoke the function (with its arguments) to refer to actual results of the function.
+
+// 🔴 **Commit your work.** 
+// The commit message should read: 
+// "Commit 1 - Verbal questions".
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+// 2. Palindrome again.
+// Write a function checkPalindrome that accepts a single argument, a string. Yes, you've done it before, but do it again. Later in this assignment we're gonna beef up our palindrome function some. See if you can do it without looking back at your previous answer. The function should return true if the string is a palindrome, false if not. Make sure your function will give the correct answer for words with capital letters.
+const checkPalindrome = (string) => {
+	// Split string
+	let stringSplit = string.split('');
+	// Reverse elements order
+	let stringReverse = stringSplit.reverse();
+	// Join back together
+	let stringJoin = stringReverse.join('').toLowerCase();
+	// Are string and reverse equal?
+	if (string.toLowerCase() === stringJoin) {
+		return true;
+	} else {
+		return false;
+	}
+}
+// //  // Also Tried this vvv Can't make it to work...
+// const checkPalindrome = (string) => {
+// 	// Just in case
+// 	console.log('string: ' + string);
+// 	// Declare array
+// 	const stringArray = [];
+// 	// Store string into array .push()
+// 	// Transform string .toLowerCase
+// 	stringArray.push(string.toLowerCase());
+// 	console.log('stringArray: ' + stringArray);
+// 	// Declare stringArrayReverse
+// 	const stringArrayReverse = [];
+// 	// Push into array the following
+// 	// Split word string into array of letter strings .split('')
+// 	// .reverse() them
+// 	// .join('') them back together
+// 	stringArrayReverse.push(string.toLowerCase().split('').reverse().join(''));
+// 	console.log('stringArrayReverse: ' + stringArrayReverse);
+
+// 	while (stringArray !== stringArrayReverse || stringArray == null || stringArrayReverse == null) {
+// 		return false;
+// 	};
+// 	return true
+// }
+
+console.log(checkPalindrome("Radar"));
+//  => true
+console.log(checkPalindrome("Borscht"));
+// => false
+
+// 🔴 **Commit your work.** 
+// The commit message should read: 
+// "Commit 2 - Palindrome".
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+// 3. Digit Sum
+// Write a function sumDigits that accepts a number and returns the sum of its digits.
+const sumDigits = (number) => {
+	// This is absolutely brilliant (I did NOT come up with this) I'm looking up the math behind it.
+	// Okay so: Modulo 9 counts the number of 10s in a number (and gives us a remainder of 1 each time)
+	return (number - 1) % 9 + 1;
+}
+console.log(sumDigits(42));
+// => 6;
```

Comment:
not sure how brilliant it is because it doesn't work
try a solution you did come up with....

Reply:
also test your code more so that you don't need me to tell you it doesn't work

Reply:
I understand.. I'll try and solve it without.

SideNote: I don't mean to stir anything up...but I do believe it works, it is however recursive and reduces any more-than-one-digit-number to a one-digit-number . 
With the code above, sumDigits(357) would become sumDigits(15) which would become 6
I would have to make a loop through the numbers go only once.
I think I'll try and do that too :|
Thank you!